lib/ansible/modules/cloud/azure/azure_rm_postgresqldatabase.py

```diff
@@ -0,0 +1,276 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2017 Zim Kalinowski, <zikalino@microsoft.com>
+#
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_postgresqldatabase
+version_added: "2.5"
+short_description: Manage PostgreSQL Database instance.
+description:
+    - Create, update and delete instance of PostgreSQL Database.
+
+options:
+    resource_group:
+        description:
+            - The name of the resource group that contains the resource. You can obtain this value from the Azure Resource Manager API or the portal.
+        required: True
+    server_name:
+        description:
+            - The name of the server.
+        required: True
+    name:
+        description:
+            - The name of the database.
+        required: True
+    charset:
+        description:
+            - The charset of the database.
+    collation:
+        description:
+            - The collation of the database.
```

Comment:
Same as charset, is there a source where I can find valid values, if not can we at least document some in the examples